src/js/themes/base.js

```diff
@@ -378,8 +378,29 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         radius: `${baseSpacing * 0.75}px`,
       },
       // color: { dark: undefined, light: undefined }
+      simple: {
+        border: {
+          // color: { dark: undefined, light: undefined }
+        },
+        // color: { dark: undefined, light: undefined }
+        hover: {
+          border: {
+            // color: {dark: undefined, light: undefined}
+          },
+          color: 'active',
```

Comment:
Is this a color or background color? and let's think if it should be `active-background`?